src/pages/signin/ValidateCodeForm/BaseValidateCodeForm.js

```diff
@@ -175,7 +176,12 @@ class BaseValidateCodeForm extends React.Component {
             formError: {},
         });
 
-        Session.signIn('', this.state.validateCode, this.state.twoFactorAuthCode);
+        const accountID = lodashGet(this.props, 'credentials.accountID');
```

Comment:
NAB: this could be inline in `Session.signInWithValidateCode(`

Reply:
Hmmm but there's an `if (accountID)`?

Reply:
oh so there is 🙈